www/csp-violation-report-endpoint.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+// Start configure
+$log_file = dirname(__FILE__) . '/csp-violations.log';
+$log_file_size_limit = 1000000; // bytes - once exceeded no further entries are added
+$email_address = 'slockwood@catchpoint.com';
```

Comment:
oh god sue i'm so sorry

Reply:
haha yeah I was going to ask.. can we use a different email